content/es/docs/concepts/storage/volume-snapshots.md

```diff
@@ -0,0 +1,151 @@
+---
+reviewers:
+- edithturn
+- raelga
+title: Volume Snapshots
```

Comment:
nit: Is this the localized title we prefer?

Reply:
> nit: Is this the localized title we prefer?

well, we didn't find any other better than this one.

Reply:
“Snapshots de Volúmenes”?